install.sh

```diff
@@ -122,3 +95,6 @@ while [ "$done" == false ]; do
   [[ ! " ${valid_input[*]} " =~ ${action} ]] && error "Invalid option"
   [[ " ${valid_input[*]} " =~ ${action} ]] && done=true && IFS=";" read -r i1 i2 <<<"${actions[$action]}" && execute "$i1" "$i2"
 done
+
+# Remove lib.sh, so next time the script is run the newest version is downloaded.
+rm -rf /tmp/lib.sh
```

Comment:
If the script fails and exists before this, the file won't be "re-downloaded". I guess that's a very niche case and probably not going to cause problems, but worth considering.

Reply:
I don't have any other solutions coming up at the moment. If the script fails, it shouldn't be run again, as it's then a not clean system and would fail in other parts. But yeah, a very niche problem.